vnext/Microsoft.ReactNative/Modules/NativeUIManager.cpp

```diff
@@ -228,7 +284,7 @@ void NativeUIManager::AddRootView(ShadowNode &shadowNode, facebook::react::IReac
   view.as<xaml::FrameworkElement>().FlowDirection(
       I18nManager::IsRTL(m_context.Properties()) ? xaml::FlowDirection::RightToLeft : xaml::FlowDirection::LeftToRight);
 
-  m_tagsToYogaNodes.emplace(rootTag, make_yoga_node(m_yogaConfig));
+  m_tagsToYogaNodes.emplace(rootTag, make_yoga_node(m_context, &YogaLog));
```

Comment:
nit: we still only have a single static YGLogger, so we don't strictly need to pass it as a param, but it shouldn't matter too much either way.

Reply:
Will leave as is for now. Not sure what YGConfigSetLogger does, but will leave this just in case anything changes in Yoga later.